routers/user.js

```diff
@@ -1,109 +0,0 @@
-const {getCurrentTimeYYYYMMDDSS} = require("../utility/time-utility");
```

Comment:
Is deleting this file on purpose? If so, what about the functions defined in the file?

Reply:
It is not used and referenced.